pkg/modulewriter/modulewriter.go

```diff
@@ -390,3 +406,37 @@ func writeExpandedBlueprint(depDir string, dc config.DeploymentConfig) error {
 
 	return nil
 }
+
+func writeDestroyInstructions(f *os.File, dc config.DeploymentConfig, deploymentDir string) {
```

Comment:
Let's take `io.Writer` instead of `os.File`, IIUC it should satisfy your requirements, while making testing easier.

Reply:
Done